application/src/main/java/org/togetherjava/tjbot/config/FreeCommandConfig.java

```diff
@@ -0,0 +1,60 @@
+package org.togetherjava.tjbot.config;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Config instance for the Free Command System see
+ * {@link org.togetherjava.tjbot.commands.free.FreeCommand}
+ *
+ * The Json looks as follows:
+ * 
+ * <pre>
+ * "freeCommand": [
+ *   {
+ *       "statusChannel": long_number,
+ *       "monitoredChannels": [long_number, long_number]
+ *   }]
+ * </pre>
+ * 
+ * Additional Guilds may add their settings by adding additional {"statusChannel....... }
+ *
+ * The long channel ID can be found by right-clicking on the channel and selecting 'Copy ID'
+ */
+@SuppressWarnings("ClassCanBeRecord")
+@JsonRootName(value = "freeCommand")
+public final class FreeCommandConfig {
+    private final long statusChannel;
+    private final List<Long> monitoredChannels;
+
+    @JsonCreator
+    private FreeCommandConfig(@JsonProperty("statusChannel") long statusChannel,
```

Comment:
why making it private? then it cant be created by code anymore, for example within a unit test.
maybe package private or public (stick to the visibility of Configs constructor id say)

Reply:
its parent (Config) is a private constructor, i just made it private for consistency

Reply:
touche, lmao... fair enough, i guess 🤷